gamemode/core/derma/cl_character.lua

```diff
@@ -534,6 +534,11 @@ end
 vgui.Register("ixCharMenu", PANEL, "EditablePanel")
 
 if (IsValid(ix.gui.characterMenu)) then
+	if (IsValid(ix.gui.characterMenu.channel)) then
+		ix.gui.characterMenu.channel:Stop()
+		ix.gui.characterMenu.channel = nil
+	end

```

Comment:
Put this in PANEL:OnRemove(), putting it there makes sure the panel always cleans up after its been removed instead of just when its auto-refreshed and is generally cleaner.

Reply:
Done.